### PR TITLE
ci: :green_heart: new pipeline to early test next version

### LINKED ISSUE #398: ci: workflow for next version
https://github.com/bikecoders/ngx-deploy-npm/blob/main/.github/workflows/next-test.yml

### PR DESCRIPTION
**Pull request by samuelfernandez**
Initial workflow to test next pipeline. TODO: optimize and parallelize

Closes #398

### CODE REVIEW

**Comment by samuelfernandez** [comment]
cc @dianjuar

File: .github/workflows/next.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+      - name: NPM
+        uses: ng-easy/npm-setup@v3.0.0
+        with:
+          nx-key: next
+
+      - name: Migrate @next
+        run: |
+          ./nx-create-migrations.sh
+          npx nx migrate next
```
Comment (dianjuar) [inline .github/workflows/next.yml:31]: `npx nx migrate next`

This is so smart. 
Way better than I'm doing.

File: .github/workflows/next.yml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+          npx nx migrate --run-migrations
```
Comment (dianjuar) [inline .github/workflows/next.yml:32]: I'm 90% secure that you need to install dependencies again once you run the migrations.

**Review by dianjuar — COMMENTED** [review]
Great Job 🧠

File: .github/workflows/next.yml (hunk 2 above)
Comment (samuelfernandez) [reply .github/workflows/next.yml:32]: Lol. In theory yes, but Nx is smart enough and does it for you. I have it working like this in my Renovate pipelines.

Still need to dig a bit more on why it fails.

Comment (dianjuar) [reply .github/workflows/next.yml:32]: `nx migrate` has a message that tells you something like 

> Review your package.json, and if everything makes sense, do `npm install`

That's why I'm saying _I'm 90% secure that you need to install dependencies again..._

Comment (samuelfernandez) [reply .github/workflows/next.yml:32]: https://github.com/nx-squeezer/squeezer/actions/runs/4099527756/jobs/7069523682

image

🤷

Comment (dianjuar) [reply .github/workflows/next.yml:32]: oh wow, it seems that it is doing it for you.

**Pull request merged**